src/Octoshift/Services/GithubApi.cs

```diff
@@ -516,7 +517,7 @@ public virtual async Task<string> GetMigrationLogUrl(string org, string repo)
 
                 var nodes = (JArray)data["data"]["organization"]["repositoryMigrations"]["nodes"];
 
-                return nodes.Count == 0 ? null : (string)nodes[0]["migrationLogUrl"];
+                return nodes.Count == 0 ? null : (MigrationLogUrl: (string)nodes[0]["migrationLogUrl"], MigrationId: (string)nodes[0]["id"]);
```

Comment:
@ArinGhazarian Please ignore the rest of my changes - but I'm stuck right now because the compiler won't let me return `null` here. What am I doing wrong? 

```
/workspaces/gh-gei/src/Octoshift/Services/GithubApi.cs(520,17): error CS8031: Async lambda expression converted to a 'Task' returning delegate cannot return a value. Did you intend to return 'Task<T>'? [/workspaces/gh-gei/src/Octoshift/Octoshift.csproj]
```

I think it should be nullable given the `Task<(string MigrationLogUrl, string MigrationId)?>` return type.